app/views/sales/_salelist.html.erb

```diff
@@ -0,0 +1,65 @@
+<% require 'date' %>
+
+<% @month_num = {"01" => 'JAN', "02" => 'FEB', "03" => 'MAR', "04" => 'APR', "05" => 'MAY', "06" => 'JUN', "07" => 'JUL', "08" => 'AUG', "09" => 'SEP', "10" => 'OCT', "11" => 'NOV', "12" => 'DEC'} %>
+
+<ul class = "groupheading">
+  <li>
+    <% v_id = params[:id] %>
+    <% used_month = @month_num[@current] %>
+    <%= link_to "CURRENT MONTH", "/sales/#{v_id}/#{used_month}", method: :get %>
+  </li>
+  <li>
+    <%="|"%>
+  </li>
+  <% @group.delete(nil) %>
+  <% @group.each do |key, value| %>
+    <li>
+      <% v_id = params[:id] %>
+      <% used_month = @month_num[key] %>
+      <%= link_to "#{used_month}", "sales/#{v_id}/#{used_month}", method: :get %>
+    </li>
+    <li>
+      <%="|"%>
```

Comment:
It seems like you probably don't need Ruby tags here since all you are doing is interpolating a string.